### DISCUSSION TITLE
Planning docenti

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#570] [Idee e proposte]
Sarebbe possibile aggiungere la possibilità che ogni docente possa inserire il proprio orario settimanale e aggiungere un planning settimanale così da sapere in che classe bisogna firmare/si ha firmato (come in altri registri come Argo)
Cordiali saluti
Prof Coco Davide

**Comment by trinko** [comment]
La segno come richiesta.
